phpunit.xml.dist

```diff
@@ -11,6 +11,7 @@
     <php>
         <ini name="memory_limit" value="-1"/>
         <ini name="apc.enable_cli" value="1"/>
+        <ini name="error_reporting" value="E_ALL ~ &amp;E_USER_DEPRECATED"/>
```

Comment:
I'm not sure this will work. I think you might need to use `16383` with a comment on where that value came from.